example/lib/pages/polygon.dart

```diff
@@ -132,12 +134,28 @@ class PolygonPage extends StatelessWidget {
                     ),
                     Polygon(
                       points: holeOuterPoints,
-                      //holePointsList: [],
+                      isFilled: true,
```

Comment:
FYI, this is one of the edge cases, where previously it would fill even though isFilled is false by default. Arguably it should have not been filled before.